### PR TITLE
EVG-2547 create script to update lobster

### PR DESCRIPTION
**Pull request by john-m-liu**

### CODE REVIEW

**Comment by tychoish** [comment]
also changing the working directory is a thing I tend to like to avoid, because it makes debugging harder.

**Comment by john-m-liu** [comment]
Made those changes. Hopefully it's less of a nightmare now

File: buildscripts/update-lobster.sh [hunk 1]
```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# this script pulls the evergreen-ci/lobster repo and copies the files so that
+# they can be served by logkeeper
+# it should only be run from the base logkeeper directory
+set -o errexit
```
Comment (bsamek) [inline buildscripts/update-lobster.sh]: it might be helpful to also `set -o xtrace`

File: buildscripts/update-lobster.sh [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+LOGKEEPER_STATIC_DIR=public/static
+LOGKEEPER_JS_DIR=js
+LOGKEEPER_CSS_DIR=css
+LOGKEEPER_TEMPLATES_DIR=templates/lobster/build
+
+# clone lobster repo and change to it
+pushd $SCRIPTS_DIR
+rm -rf $LOBSTER_DIR
+git clone $LOBSTER_REPO $LOBSTER_DIR
+pushd .lobster-temp
```
Comment (bsamek) [inline buildscripts/update-lobster.sh]: use `cd` here, so you only need to pop once

File: buildscripts/update-lobster.sh [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+# build lobster
+npm install
+npm run build
+
+# replace existing js/css files in logkeeper with the updated ones
+popd && popd
```
Comment (bsamek) [inline buildscripts/update-lobster.sh]: only pop once after `cd` change on 23

File: buildscripts/update-lobster.sh (hunk 2 above)
Comment (tychoish) [reply buildscripts/update-lobster.sh]: also this is LOBSTER_DIR

Comment (tychoish) [reply buildscripts/update-lobster.sh]: I'd actually leave the pushd, and rearrange the pushes so they're at the beginning of the blocks, and pops so their at the end.

File: buildscripts/update-lobster.sh (hunk 1 above)
Comment (john-m-liu) [reply buildscripts/update-lobster.sh]: done

File: buildscripts/update-lobster.sh (hunk 2 above)
Comment (john-m-liu) [reply buildscripts/update-lobster.sh]: went with Sam's suggestion

File: buildscripts/update-lobster.sh [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+# clone lobster repo
+pushd $SCRIPTS_DIR
+rm -rf $LOBSTER_DIR
+git clone $LOBSTER_REPO $LOBSTER_DIR
+# build lobster
+pushd $LOBSTER_DIR
+npm install
+npm run build
+popd && popd
```
Comment (tychoish) [inline buildscripts/update-lobster.sh]: I'd just put them on seperate lines

**Pull request merged**